### PR TITLE
feat: Collateralization thresholds upgrade

### PR DESCRIPTION
**Pull request by savudani8**

### CODE REVIEW

File: parachain/src/chain_spec.rs [hunk 1]
```diff
@@ -405,18 +405,36 @@ fn testnet_genesis(
...
-            )], /* 135% */
-            liquidation_collateral_threshold: vec![(
-                default_pair(CurrencyId::KSM),
-                FixedU128::checked_from_rational(110, 100).unwrap(),
-            )], /* 110% */
+            secure_collateral_threshold: vec![
+                (
+                    default_pair(CurrencyId::DOT),
+                    FixedU128::checked_from_rational(330, 100).unwrap(),
+                ), /* 330% */
```
Comment (gregdhill) [inline parachain/src/chain_spec.rs:412]: Remove DOT values for testnet genesis.

File: parachain/src/chain_spec.rs [hunk 2]
```diff
@@ -616,18 +634,36 @@ fn kintsugi_mainnet_genesis(
...
-            )], /* 200% */
-            liquidation_collateral_threshold: vec![(
-                default_pair(CurrencyId::KSM),
-                FixedU128::checked_from_rational(150, 100).unwrap(),
-            )], /* 150% */
+            secure_collateral_threshold: vec![
+                (
+                    default_pair(CurrencyId::DOT),
+                    FixedU128::checked_from_rational(330, 100).unwrap(),
+                ), /* 330% */
```
Comment (gregdhill) [inline parachain/src/chain_spec.rs:641]: Remove DOT values for Kintsugi genesis.

File: parachain/src/chain_spec.rs [hunk 3]
```diff
@@ -823,18 +859,36 @@ fn interlay_mainnet_genesis(
...
+                (
+                    default_pair(CurrencyId::KSM),
+                    FixedU128::checked_from_rational(360, 100).unwrap(),
+                ), /* 360% */
```
Comment (gregdhill) [inline parachain/src/chain_spec.rs:870]: Remove KSM values for Interlay genesis.

**Review by gregdhill — CHANGES_REQUESTED** [review]

**Review by gregdhill — DISMISSED** [review]

**Review by gregdhill — APPROVED** [review]

**Comment by nud3l** [comment]
@savudani8 please add a quick summary of the parameter changes in future PRs. So it's easier to check how the parameters should be changed. Thanks!

File: parachain/src/chain_spec.rs [hunk 4]
```diff
@@ -613,21 +620,21 @@ fn kintsugi_mainnet_genesis(
...
-                                                                                                            * USD at
-                                                                                                            * time of writing */
+            system_collateral_ceiling: vec![(default_pair_kintsugi(CurrencyId::KSM), 317 * CurrencyId::KSM.one())], /* 317 ksm, about 100k
+                                                                                                                     * USD at
+                                                                                                                     * time of writing */
             secure_collateral_threshold: vec![(
-                default_pair(CurrencyId::KSM),
-                FixedU128::checked_from_rational(260, 100).unwrap(),
-            )], /* 260% */
+                default_pair_kintsugi(CurrencyId::KSM),
```
Comment (nud3l) [inline parachain/src/chain_spec.rs:627]: @savudani8 @gregdhill why are the thresholds so heavily increased? We need to discuss these parameter changes before making changes. There is a risk/product trade-off here. Increasing the collateralization by 100% points is reducing the profitability of running a vault significantly.

**Pull request merged**